sdk/build.gradle

```diff
@@ -30,3 +30,10 @@ animalsniffer {
             sourceSets.main
     ]
 }
+
+processResources {
+    new File(
```

Comment:
I don't think this is doing what you expect, this code is run during the Gradle configuration, not during any task which is not good for several reasons.

Instead, add a new task and set it as a `builtBy` source

https://github.com/aws/aws-xray-sdk-java/blob/master/build.gradle.kts#L63
https://github.com/aws/aws-xray-sdk-java/blob/master/build.gradle.kts#L131